extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -149,6 +150,13 @@ export const portMustBeNumber = localize('portMustNotBeNumber', "Port must a be
 export const valueCannotBeEmpty = localize('valueCannotBeEmpty', "Value cannot be empty");
 export const dockerImageLabelPrefix = 'source=sqldbproject';
 export const dockerImageNamePrefix = 'sqldbproject';
+
+export const eulaAgreementTemplate = localize('eulaAgreementTemplate', "I accept the {0}.");
```

Comment:
Would suggest adding a comment (passed as a parameter to the localize function) saying that the placeholder is left there on purpose and should not be translated.